_posts/2020-04-12-final-presentations-youre-all-invited.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Graduation and final presentations - you're all invited!
+authors:
+- yochannah
+- malvikasharan
+- bebatut
+image: https://images.unsplash.com/photo-1560523159-6b681a1e1852?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2100&q=80
+---
+
+It's been a long and exciting journey, from applications opening in October 2019, [announcing selected applicants in early January](https://openlifesci.org/posts/2020/01/13/ols1-announcement/), and kicking off [the program's first cohort call on the 29th of January](https://openlifesci.org/ols-1/week02/). Since then we've had an amazing range of guest speakers from around the world, sharing their expertise on topics in [the OLS syllabus](https://openlifesci.org/ols-1), as well as a variety of breakout and discussion sessions, interspersed with mentor calls every other week. We wrapped up our learning calls with [a cohort call on self care, ally skills, and careers](https://openlifesci.org/ols-1/week11/).  
```

Comment:
```suggestion
It's been a long and exciting journey, from applications opening in October 2019, [announcing selected applicants in early January]({% link _posts/2020-01-13-ols1-announcement.md %}), and kicking off [the program's first cohort call on the 29th of January]({% link _ols-1/week02.md %}). Since then we've had an amazing range of guest speakers from around the world, sharing their expertise on topics in [the OLS syllabus]({% link ols-1.md %}), as well as a variety of breakout and discussion sessions, interspersed with mentor calls every other week. We wrapped up our learning calls with [a cohort call on self care, ally skills, and careers]({% link _ols-1/week11.md %}).  
```

Reply:
In case we change the URL at some point